src/Razor/src/Microsoft.VisualStudio.Editor.Razor/DefaultVisualStudioRazorParser.cs

```diff
@@ -52,8 +50,6 @@ internal DefaultVisualStudioRazorParser(RazorCodeDocument codeDocument)
         public DefaultVisualStudioRazorParser(
             ForegroundDispatcher dispatcher,
             VisualStudioDocumentTracker documentTracker,
-            ProjectSnapshotProjectEngineFactory projectEngineFactory,
```

Comment:
This is no longer built from Roslyn services so it can't be constructed with these. It needed to be constructable without a workspace.